src/pages/Home/Home.jsx

```diff
@@ -1,3 +1,48 @@
+import "./Home.css";
+import axios from "axios";
+import BannerImg1 from "../../assets/BannerImage1.jpg";
+import BannerImg2 from "../../assets/BannerImage2.jpg";
+import { useEffect, useState } from "react";
+import { Category, Banner, Loading } from "../../components/index";
+
 export const Home = () => {
-  return <div>Home Page</div>;
+  const [categories, setCateories] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const fetchData = async () => {
+    try {
+      setLoading(true);
+      const { data } = await axios.get("/api/categories");
+      setCateories(data.categories);
+    } catch (error) {
+      setError(error);
+    } finally {
+      setLoading(false);
+    }
+  };
+  useEffect(() => {
+    fetchData();
+  }, []);
+  return (
```

Comment:
can use getAxios custom hook here for get requests

Reply:
Thanks, Raman planning to that